src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php

```diff
@@ -98,6 +98,32 @@ protected function getRequest(SentMessage $message): SendEmailRequest
         if ($header = $email->getHeaders()->get('X-SES-SOURCE-ARN')) {
             $request['FromEmailAddressIdentityArn'] = $header->getBodyAsString();
         }
+        if ($email->getHeaders()->has('X-SES-LIST-MANAGEMENT-OPTIONS') || $email->getHeaders()->has('X-SES-CONTACT-LIST-NAME')) {
+            $input = [
+                'ContactListName' => null,
+            ];
+            
+            if ($email->getHeaders()->get('X-SES-LIST-MANAGEMENT-OPTIONS')) {
+                $options = $email->getHeaders()->get('X-SES-LIST-MANAGEMENT-OPTIONS')->getBodyAsString();
+                $options = explode('; topic=', $options);
+    
+                $input['ContactListName'] = $options[0];
+    
+                if (count($options) > 1) {
+                    $input['TopicName'] = $options[1];
+                }
+            } else {
+                $input['ContactListName'] = $email->getHeaders()->get('X-SES-CONTACT-LIST-NAME')->getBodyAsString();
+                
+                if ($header = $email->getHeaders()->get('X-SES-TOPIC-NAME')) {
+                    $input['TopicName'] = $header->getBodyAsString();
+                }
+            }
+            
+            if ($input['ContactListName']) {
+                $request['ListManagementOptions'] = new ListManagementOptions($input);
```

Comment:
The class does not exist

Reply:
@tfavareille Any news?

Reply:
missing use for `https://github.com/async-aws/aws/blob/master/src/Service/Ses/src/ValueObject/ListManagementOptions.php`?